src/subclass/container.rs

```diff
@@ -109,6 +114,16 @@ impl<T: ContainerImpl + ObjectImpl> ContainerImplExt for T {
             from_glib_none(f(container.to_glib_none().0, widget.to_glib_none().0))
         }
     }
+
+    fn parent_forall<P: FnMut(&Widget)>(&self, container: &Container, callback: P) {
+        unsafe {
+            let data = self.get_type_data();
+            let parent_class = data.as_ref().get_parent_class() as *mut gtk_sys::GtkContainerClass;
+            if let Some(f) = (*parent_class).forall {
+                f(container.to_glib_none().0, false, callback.to_glib_none().0)
+            }
+        }
+    }
 }
 
 unsafe impl<T: ObjectSubclass + ContainerImpl> IsSubclassable<T> for ContainerClass {
```

Comment:
You also need to add it here

Reply:
Done - The `include_internals` param should be added everywhere now.